spec/lib/tito_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "climate_control"
+require "json"
+require "webmock/rspec"
+require "tito"
+
+describe Tito do
+  around do |example|
+    ClimateControl.modify TITO_AUTH_TOKEN: 'bar' do
+      example.run
+    end
+  end
+
+  describe "#event_list" do
+    it "returns a list of events" do
+      event_data = { "title" => "event info" }
```

Comment:
Stubbing out the request for the failed one works well, why not do the same here?